### DISCUSSION TITLE
How to charge for a secondary (or subsequent) newsletter if the first is free?

### DISCUSSION THREAD

**Discussion by JPFlintoff** [buttondown/discussions#48] [Q&A] [answered]
Hello. I originally made payment optional for my main Buttondown newsletter. But since nobody was paying I stopped, thinking I might charge for a pop-up series I had planned - for anybody who chose to opt in.

If this is easy, I'd love to know how. 

Otherwise, I guess I can ask people to pay me first and then manually "gift" them access to the paid letter... I don't have so many readers that this is a problem (yet, he added hopefully).

JP

**Comment by jmduke** [comment, marked as answer]
@JPFlintoff hello hello! This should be possible just 'out of the box' — I'm wondering if there's a nuance I'm missing. You should be able to just give them a CTA to subscribe to the new newsletter in the body of the first one. (Payment Links in particular might be very useful here!)
